sites/platform/src/dedicated-gen-3/options.md

```diff
@@ -23,4 +33,17 @@ But note that multiple-AZ configurations don't improve the contractual 99.99% up
 {{% vendor/name %}} is responsible for meeting the 99.99% uptime SLA no matter what,
 so multiple-AZ deployments should only be considered in cases where they're truly appropriate.
 
-Multi-AZ deployments are available only on select AWS regions.
\ No newline at end of file
+Multi-AZ deployments are available only on select AWS regions.
+
+## `sftp`
+
+In addition to SSH accounts, you can create `sftp` accounts with a custom user/password.
```

Comment:
similar to the other comment: should we link from this section over to /development/file-transfer.html#sftp?